subprojects/packagefiles/x265/source/test/meson.build

```diff
@@ -0,0 +1,75 @@
+have_rdtsc = cc.has_header_symbol('intrin.h', '__rdtsc')
+if have_rdtsc
+  c_args += '-DHAVE_RDTSC=1'
+elif host_arch.startswith('x86')
+  # FIXME: On AppleClang x86_64 __rdtsc is a builtin --Amy
+  if cc.compiles(
+    'int main() { return __rdtsc(); }',
+    name: 'Has __rdtsc as a builtin',
+  )
+    c_args += '-DHAVE_RDTSC=1'
+    # Cheats forced inclusion of intrin.h with HAVE_RDTSC
+    include_dirs += include_directories('../meson')
+  endif
+endif
+
+# add X86 assembly files
+if host_arch.startswith('x86')
+  nasm_src = files('checkasm-a.asm')
+  # add ARM assembly files
+elif host_arch == 'arm'
+  nasm_src = files('checkasm-arm.S')
+else
+  nasm_src = []
+endif
+
+testbench_srcs = nasm_src + files(
+  'intrapredharness.cpp',
+  'intrapredharness.h',
+  'ipfilterharness.cpp',
+  'ipfilterharness.h',
+  'mbdstharness.cpp',
+  'mbdstharness.h',
+  'pixelharness.cpp',
+  'pixelharness.h',
+  'testbench.cpp',
+  'testharness.h',
+)
+
+# FIXME: on CMake checkasm-arm uses NASM_FLAGS,
```

Comment:
Also this FIXME, sounds like it should just be a normal comment?

Reply:
I marked these as FIXMEs because they need to be addressed by the upstream maintainers; it's not a bug on our end, but here we have the advantage of being able to case them out.

The particular bug here is that this line mixes NASM and GAS flags thus breaking the build on Arm64.

Reply:
Yeah no, we should not put FIXMEs in our code for things that other people have to fix, it's very misleading. Please remove it :)